src/robot/conf/settings.py

```diff
@@ -607,3 +618,7 @@ def console_output_config(self):
     @property
     def process_empty_suite(self):
         return self['ProcessEmptySuite']
+
+    @property
+    def expand_keywords_args(self):
```

Comment:
The `_args` postfix isn't needed. Others don't have anything like that either.

Reply:
fixed